azure-iot-hub-devicesdk/azure/iot/hub/devicesdk/transport/mqtt/mqtt_transport.py

```diff
@@ -384,21 +385,39 @@ def send_output_event(self, message, callback=None):
     def _on_shared_access_string_updated(self):
         self._trig_on_shared_access_string_updated()
 
-    def enable_input_messages(self, callback=None, qos=1):
+    def enable_feature(self, feature_name, callback=None, qos=1):
```

Comment:
>enable_feature [](start = 8, length = 14)

again I understand why we do this but i would like to validate this with our user study, because we're the only SDK doing this and it feels like additional hoops and if we have a way to auto-enable we should.

Reply:
We are autoenabling as far as the client is concerned. Not sure what this comment means. Will discuss further at a later time.

---
In reply to: [259538058](https://github.com/Azure/azure-iot-sdk-python-preview/pull/49#discussion_r259538058) [](ancestors = 259538058)